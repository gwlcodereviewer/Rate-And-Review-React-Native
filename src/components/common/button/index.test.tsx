import React from 'react';
import renderer from 'react-test-renderer';
import PrimaryButton from '.';

let wrapper: renderer.ReactTestRenderer;

describe('render correctly', () => {
  wrapper = renderer.create(<PrimaryButton title='Submit' onPress={() => {}}/>);

  it('render correctly primaryButton view', () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
