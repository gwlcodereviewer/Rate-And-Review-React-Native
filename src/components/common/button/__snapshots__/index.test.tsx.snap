// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly render correctly primaryButton view 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#1778F2",
      "borderColor": "#1778F2",
      "borderRadius": 6,
      "borderWidth": 1,
      "height": 57.5,
      "justifyContent": "center",
      "opacity": 1,
      "paddingHorizontal": 7.5,
      "paddingVertical": 7.5,
      "width": "100%",
    }
  }
>
  <Text
    allowTextTransform={true}
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Flink-Bold",
          "fontSize": 16.5,
          "lineHeight": 34.5,
          "textTransform": "capitalize",
        },
      ]
    }
  />
</View>
`;
